### ISSUE TITLE
Could we use samba static lib in ios app ? Samba is under GPL ?

### ISSUE THREAD

**Issue by James24** [kolyvan/kxsmb#29]
If i use samba client in ios , app store wont reject the app due to GPL  license of samba or is there any trick to submit it app store ? ? @kolyvan @alex-v1981

**Comment by pantunes** [comment]
There are some iPhone apps in App Store that open smb shares, so I assume that should be no problem.
Examples:
https://itunes.apple.com/en/app/fileexplorer-free/id510282524?mt=8
https://itunes.apple.com/us/app/filebrowser-for-business/id854618029?mt=8

**Comment by James24** [comment]
Yes @pantunes there are many app using smb share. But we dono they using it from samba.com Thanks for your contribution. Let we see others opinions :+1:
